src/components/footer/_footer.scss

```diff
@@ -30,12 +30,14 @@
         vertical-align: middle;
     }
 
-    &__poweredBy-link {
-        &:focus {
-            .ons-icon--logo {
-                @extend %a-focus;
-            }
-        }
+    &__extra-logo svg,
+    img {
+        max-height: 30px;
+        width: auto;
+    }
+
+    &__logo {
+        gap: 1rem;
```

Comment:
I think the names of these classes is confusing, these classes are called `_logo` and `__extra-logo` but these aren't actually set on the logos themselves. The css in `__logo` class isn't being applied to the logo either, this is setting the gap on the logo container. You could restructure this so that there was one class called `_logo-container` set the gap on that and then within that select the img or svg from there